meson_options.txt

```diff
@@ -0,0 +1 @@
+option('build_tests', type : 'boolean', value : true)
```

Comment:
It'd be nice if this could be a feature type defaulting to 'auto' rather than a boolean (just call it 'tests'). Then you can pass the option to the 'required' field of the gmock/gtest dependency objects.